src/pages/NewTaskPage.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import withLocalize, {withLocalizePropTypes} from '../components/withLocalize';
+import compose from '../libs/compose';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import Navigation from '../libs/Navigation/Navigation';
+import ScreenWrapper from '../components/ScreenWrapper';
+import styles from '../styles/styles';
+import ONYXKEYS from '../ONYXKEYS';
+import * as ErrorUtils from '../libs/ErrorUtils';
+import Form from '../components/Form';
+import TextInput from '../components/TextInput';
+import Permissions from '../libs/Permissions';
+
+const propTypes = {
+    /** List of betas available to current user */
+    betas: PropTypes.arrayOf(PropTypes.string),
+
+    ...withLocalizePropTypes,
+};
+const defaultProps = {
+    betas: [],
+};
+
+const NewTaskPage = (props) => {
+    /**
+     * @param {Object} values - form input values passed by the Form component
+     * @returns {Boolean}
+     */
+    function validate(values) {
```

Comment:
Here we could use the arrow method instead.

NAB

Reply:
Added comment in the component that this is just placeholder stuff until the next issue comes along so going to leave it to be addressed later